bootstrap.py

```diff
@@ -124,9 +125,15 @@ def delete_directory(directoryname):
         sys.exit(1)
 
 
-def yum(command, pkgs=""):
-    """Helper function to call a yum command on a list of packages."""
-    exec_failexit("/usr/bin/yum -y %s %s" % (command, pkgs))
+def call_yum(command, pkgs="", warnonerror=False):
```

Comment:
I would have called the param `failonerror` and default it to `True` as the name `warnonerror` somehow implies that otherwise nothing happens on error.
Also, while we edit this function already, can we rename `pkgs` to `params` or something? when we call not install/remove, the pkgs name does not really match.